### ISSUE TITLE
Event handling related memory leak

### ISSUE THREAD

**Issue by salviati** [salviati/go-qt5#13]
> If you go run minimal.go and keep resizing the window for a while, you'll see that the memory usage goes up.

(from http://github.com/visualfc/go-ui/issues/10)

**Comment by phaikawl** [comment]
I have tried what he said and see no consistent leaking behavior, the memory usage randomly increases and randomly drops, no real consistent changes, after a while it's pretty much the same number.
Is there really a clear evidence? Have you tested it thoroughly?

**Comment by salviati** [comment]
> Is there really a clear evidence? Have you tested it thoroughly?
No; as I mentioned in README, I don't have any time to work actively on this project.
Have you? It may be restricted to a certain type event.

**Comment by phaikawl** [comment]
I have tried what he said and see no consistent leaking behavior, the virtual memory usage randomly increases and randomly drops, no real consistent changes, after a while it's pretty much the same number.

**Comment by salviati** [comment]
I remember checking this with minimal.go 2 years ago, and I observed the increasing memory usage. I'm not sure what has changed in go/qt5 since then, but I can't seem to reproduce it now.

I'll keep this issue open until we have a test for this. Thanks for testing it!

**Comment by phaikawl** [comment]
I guess it's somehow automatically fixed by either the updated Qt or the updated cgo then, that could only be good news :smile: .
Also, what do you mean by "a test"? How is it supposed to be created? Do you mean having valgrind available for cgo or something?

**Comment by salviati** [comment]
A test that programmatically creates a lot events and watches the memory usage via procfs, you wouldn't need valgrind or cgo for that.
